packages/legend-query/src/components/QueryBuilderFunctionsExplorerPanel.tsx

```diff
@@ -0,0 +1,588 @@
+      !isDragging ||
+      !item ||
+      !Object.values(QUERY_BUILDER_FUNCTION_TREE_DND_TYPE).includes(itemType) ||
+      item.node.dndType === QUERY_BUILDER_FUNCTION_TREE_DND_TYPE.PACKAGE
+    ) {
+      return null;
+    }
+    return (
+      <div className="query-builder__functions-explorer__tree__drag-preview-layer">
+        <div
+          className="query-builder__functions-explorer__tree__drag-preview"
+          style={
+            !currentPosition
+              ? { display: 'none' }
+              : {
+                  transform: `translate(${currentPosition.x + 20}px, ${
+                    currentPosition.y + 10
+                  }px)`,
+                }
+          }
+        >
+          {generateFunctionSignature(
+            item.node.packageableElement as ConcreteFunctionDefinition,
+            true,
+          )}
+        </div>
+      </div>
+    );
+  },
+);
+
+const QueryBuilderSimplePackableElementTreeNodeContainer = observer(
+  (props: {
+    queryBuilderState: QueryBuilderState;
+    element: ConcreteFunctionDefinition;
+    rootPackageName: ROOT_PACKAGE_NAME;
+  }) => {
+    const { queryBuilderState, element, rootPackageName } = props;
+    const node = generateFunctionsExplorerTreeNodeData(
+      queryBuilderState,
+      element,
+      rootPackageName,
+    );
+    const [, dragConnector, dragPreviewConnector] = useDrag(
+      () => ({
+        type: QUERY_BUILDER_FUNCTION_TREE_DND_TYPE.FUNCTION,
+        item: { node: node },
+      }),
+      [node],
+    );
+    // hide default HTML5 preview image
+    useEffect(() => {
+      dragPreviewConnector(getEmptyImage(), { captureDraggingState: true });
+    }, [dragPreviewConnector]);
+
+    return (
+      <div
+        className="query-builder__functions-explorer__function"
+        ref={dragConnector}
+      >
+        <div className="query-builder__functions-explorer__function__content">
+          <div className="query-builder__functions-explorer__function__icon">
+            <div className="query-builder__functions-explorer__function-icon">
+              <PURE_FunctionIcon />
+            </div>
+          </div>
+          <div className="query-builder__functions__function__label">
+            {generateFunctionSignature(element, true)}
+          </div>
+        </div>
+        <div className="query-builder__functions-explorer__function__actions">
+          <div className="query-builder__functions-explorer__function__action">
+            <QueryBuilderFunctionInfoTooltip
+              element={node.packageableElement as ConcreteFunctionDefinition}
+            >
+              <div className="query-builder__functions-explorer__function__action query-builder__functions-explorer__function__node__info">
+                <InfoCircleIcon />
+              </div>
+            </QueryBuilderFunctionInfoTooltip>
+          </div>
+        </div>
+      </div>
+    );
+  },
+);
+
+const QueryBuilderFunctionExplorerTreeNodeContainer = observer(
```

Comment:
You should be consistent and scan for all `functionExplorer` and rename them to `functionsExplorer`